g_docs.go

```diff
@@ -403,6 +404,9 @@ func analyseControllerPkg(localName, pkgpath string) {
 func isSystemPackage(pkgpath string) bool {
 	goroot := os.Getenv("GOROOT")
 	if goroot == "" {
+		goroot = runtime.GOROOT()
+	}
+	if goroot == "" {
```

Comment:
@radioinmyhead Can you remove the second `if` statement as it's no longer needed?